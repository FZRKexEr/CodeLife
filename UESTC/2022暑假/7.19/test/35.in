10
icavviwmuhprypekqnkmoecwkzazdkyvtpdxotmstdmjypxldchromxgdogpqydiyzakkoxjkoiplaxbwsyfqywpbpczxozginuh
9
pmyrghnykwpjkupgqrjzophknvouypnmeqpfbissvxxhpipeczqihpfeevutakhcqethwskjfreognjeuydldkbqttmnpzokryzd
5
zcxypxrrcbiebcfqtacwzmlvsnraqtpsqynbkjgbjaxvlehqbtkzaurljcuuaamizmvzvuhlgpsrzrenajkivawayxslutyzogke
9
fdmnqtkbpmmowdifjdbczmtesbzunfxrebtfjivfjlmkykicjhgwxctrspxixtdbdkwhfgruvnnjfkvuemoseqvytisfbmwqhzvs
3
nqcplstwfldaadsbwhhoidccejqnjyxninadfokyqvgwytsvqrcghuyezglnntczpdrerjuwbcfclokhbprjbnwqvlwilhvolxme
6
wmorlpnartblcofndxpqyqopjhbabtzlldxiobhwnfnystswbaybvsukpgxefeujwflrcopmcghbbtnzyxrwmgrmzuqtripcxrbq
7
ljivimgxfslvdecuicezljxyxpzuidrozvvbtjktkwuhuwxzxoblhsyrbjkipokywtawelohaykhcxuxhpnobaxsivmyjfoozpja
6
pgjvhmpumutgzkyroynlxcjpzsdrfadgkvtfqughmuvagqwykglcoxtygmzttcvbcviytlhemsrieszzraqyzasnlskwpysegddt
4
xqyxekzvsrrefnqdvljbiryiquqvnzhcwrlouwpgeyihcvvuxnpfcxbhsenopyquzljunvleuppluorhcwqsoirqzpurlutytntq
10
ngfthymvgzlpjfptivukqswxyehelincjcnnjpchfljumnebmugivugwlxllmsnptvsnmofouqodogesqaeukskchflmxwpsopxc
1
