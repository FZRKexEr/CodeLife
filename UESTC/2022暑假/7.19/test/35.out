2048
1048576
2048
603979768
65536
16384
65536
33554432
1024
882499718
